lib/src/main/AndroidManifest.xml

```diff
@@ -16,8 +16,9 @@
         <activity
             android:name="com.tubitv.media.activities.DoubleViewTubiPlayerActivity"
             android:configChanges="keyboard|keyboardHidden|orientation|screenSize|screenLayout|smallestScreenSize|uiMode"
-            android:launchMode="singleTop"
-            android:theme="@style/TubiPlayerTheme" />
+            android:launchMode="singleTask"
```

Comment:
Why change  launchMode of DoubleViewTubiPlayerActivity  from singleTop to singleTask?

When player is in PIP mode and some content is playing, open tubi player app by clicking app icon, DoubleViewActivity is not at top of stack at this moment.
Click Default UI or Custom UI, another instance of DoubleViewTubiPlayerActivity will be created and two videos are playing at the same time.